app/src/main/java/com/lastaoutdoor/lasta/ui/screen/profile/ProfileScreen.kt

```diff
@@ -232,52 +232,7 @@ fun ChangeBio(
 ) {
 
   if (isEditBio) {
-    val maxCharCount = 70
-    var text by remember { mutableStateOf(bioText) }
-
-    ModalBottomSheet(
-        onDismissRequest = { onDismissRequest() },
-        modifier = Modifier.fillMaxWidth().wrapContentHeight().testTag("ProfileModal")) {
-          Column(modifier = Modifier.fillMaxWidth().padding(16.dp)) {
-            Text("Bio" /* TODO : make this a xml string */)
-            Spacer(modifier = Modifier.height(8.dp))
-            TextField(
-                value = text,
-                onValueChange = {
-                  if (it.length <= maxCharCount) {
-                    text = it
-                  }
-                },
-                placeholder = { Text("Enter your bio...") /* TODO : make this a xml string */ },
-                singleLine = false,
-                maxLines = 2,
-                keyboardOptions = KeyboardOptions.Default.copy(imeAction = ImeAction.Done),
-                trailingIcon = {
-                  if (text.isNotEmpty()) {
-                    Text(
-                        "${text.length}/$maxCharCount",
-                        style = MaterialTheme.typography.bodySmall,
-                        color = if (text.length == maxCharCount) Color.Red else Color.Gray)
-                  }
-                },
-                modifier = Modifier.fillMaxWidth().testTag("ProfileTextField"))
-
-            Spacer(modifier = Modifier.height(16.dp))
-            Box(
-                contentAlignment = Alignment.Center,
-                modifier = Modifier.fillMaxWidth().testTag("ProfileBox")) {
-                  Row {
-                    Button(onClick = onDismissRequest) {
-                      Text("Cancel" /* TODO : make this a xml string */)
-                    }
-                    Spacer(modifier = Modifier.width(16.dp))
-                    Button(onClick = { onBioChange(text) }) {
-                      Text("Save" /* TODO : make this a xml string */)
-                    }
-                  }
-                }
-          }
-        }
+    BottomSheetProfile(onDismissRequest, bioText, onBioChange)
```

Comment:
The previous version was way too long and it was the right call to extract it in a composable